src/utils/hooks/useLoadReview.tsx

```diff
@@ -0,0 +1,35 @@
+import { useEffect, useState } from 'react'
+import { useGetReviewQuery } from '../generated/graphql'
+
+interface UseLoadReviewProps {
+  reviewId: number
+}
+
+const useLoadReview = ({ reviewId }: UseLoadReviewProps) => {
+  const [loading, setLoading] = useState(true)
+  const [error, setError] = useState('')
+
```

Comment:
Would you mind putting in a TO-DO here to use `useTriggerProcessing` here before loading the rest (similar to `useLoadApplication`)? I'll do the TriggerProcessing adjustments next, so this will be able to be implemented after that.

Reply:
No problem

Reply:
Ah, actually I think that will be better to be done in the next PR, just so it won't cause more conflicts